riscv/triggers.cc

```diff
@@ -25,6 +25,7 @@ reg_t disabled_trigger_t::tdata1_read(const processor_t * const proc) const noex
 void disabled_trigger_t::tdata1_write(processor_t * const proc, const reg_t val) noexcept
 {
   auto xlen = proc->get_xlen();
+  assert(get_field(val, CSR_TDATA1_TYPE(xlen)) == CSR_TDATA1_TYPE_DISABLED);
```

Comment:
This assertion is firing whenever I write `tdata1` with an unsupported `type` field.